oss_scripts/pip_package/setup.py

```diff
@@ -74,18 +74,18 @@ def finalize_options(self):
     distclass=BinaryDistribution,
     install_requires=[
         (
-            'tensorflow>=2.12.0, <2.13; platform_machine != "arm64" or'
+            'tensorflow>=2.13.0, <=2.13; platform_machine != "arm64" or'
             ' platform_system != "Darwin"'
         ),
         (
-            'tensorflow-macos>=2.12.0, <2.13; platform_machine == "arm64" and'
+            'tensorflow-macos>=2.12.0, <=2.13; platform_machine == "arm64" and'
```

Comment:
Does this >= need to be modified to 2.13 as well?  Maybe all the `<=2.13` should be `<2.14`